README.md

```diff
@@ -362,20 +362,20 @@ login will succeed.
 
 ## Troubleshooting
 
-### Obtaining a Certificate Error from the PCF API
+### Obtaining a Certificate Error from the CF API
 
 When configuring this plugin, you may encounter an error like:
 ```
-Error writing data to auth/pcf/config: Error making API request.
+Error writing data to auth/cf/config: Error making API request.
 
-URL: PUT http://127.0.0.1:8200/v1/auth/pcf/config
+URL: PUT http://127.0.0.1:8200/v1/auth/cf/config
 Code: 500. Errors:
 
 * 1 error occurred:
 	* unable to establish an initial connection to the PCF API: Could not get api /v2/info: Get https://api.sys.lagunaniguel.cf-app.com/v2/info: x509: certificate signed by unknown authority
```

Comment:
In the documentation PCF is embedded in an error message example.  This should be updated since the actual error message was updated in this PR.

Reply:
Good eye! Updating.